flake.nix

```diff
@@ -0,0 +1,23 @@
+{
+  description = "Home Manager for Nix";
+
+  outputs = { self, nixpkgs }: rec {
+
+    nixosModules.home-manager = import ./nixos;
+
+    lib = {
+      homeManagerConfiguration = { configuration, system, homeDirectory
+        , username
+        , pkgs ? builtins.getAttr system nixpkgs.outputs.legacyPackages
+        , check ? true }@args:
+        import ./modules nixpkgs {
+          pkgs = builtins.getAttr system nixpkgs.outputs.legacyPackages;
```

Comment:
Shouldn't this be `inherit pkgs` since pkgs is passed in?

Reply:
Fair point! I didn't write and don't use this part :D